### PR TITLE
[IOPAY-400] Payment Manager product

### PR DESCRIPTION
**Pull request by infantesimone**
### List of changes

- add new apim product _Payment Manager_ with the following API:
  - `Payment Manager restapi API` 
  - `Payment Manager restapi CD API`
  - `Payment Manager CD Webview`
  - `Payment Manager logging API`
  - `Payment Manager restapi RTD API`
  - `Payment Manager WISP`
  - `Payment Manager Admin Panel`      

### Motivation and context

The goal of this PR is to configure some _Payment Manager APIs_ on the _pagoPA apim_.

### Type of changes

- [x] Add new resources
- [ ] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [x] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [ ] No

### Other information

---

### CODE REVIEW

File: src/apim_payment_manager.tf [hunk 1]
```diff
@@ -0,0 +1,541 @@
...
+
+  source = "git::https://github.com/pagopa/azurerm.git//api_management_api?ref=v1.0.16"
+
+  name                  = format("%s-pm-restapi-api", local.project)
+  api_management_name   = module.apim.name
+  resource_group_name   = azurerm_resource_group.rg_api.name
+  product_ids           = [module.apim_payment_manager_product.product_id]
+  subscription_required = local.apim_pm_restapi_api.subscription_required
+  version_set_id        = azurerm_api_management_api_version_set.pm_restapi_api.id
+  api_version           = "v4"
```
Comment (pasqualespica) [inline src/apim_payment_manager.tf:109]: why directly `v4` and not `v1` ?

File: src/apim_payment_manager.tf [hunk 2]
```diff
@@ -0,0 +1,541 @@
...
+
+#####################################
+## API restapi RTD                  ##
+#####################################
+locals {
+  apim_pm_restapirtd_api = {
+    # params for all api versions
+    display_name          = "Payment Manager restapi RTD API"
+    description           = "API to support 'Registro Transazioni Digitali'"
+    path                  = "payment-manager/restapi-rtd"
```
Comment (pasqualespica) [inline src/apim_payment_manager.tf:247]: update https://pagopa.atlassian.net/wiki/spaces/TKM/pages/459309751/Configurazione+API+Payment+Manager+su+APIM+pagoPA

File: src/apim_payment_manager.tf [hunk 3]
```diff
@@ -0,0 +1,541 @@
...
+}
+
+############################
+## API webview-cd         ##
+############################
+locals {
+  apim_pm_webview_api = {
+    display_name          = "Payment Manager CD webview"
+    description           = "Webview to support app IO payments"
+    path                  = "payment-manager/webview-cd"
```
Comment (pasqualespica) [inline src/apim_payment_manager.tf:385]: update https://pagopa.atlassian.net/wiki/spaces/TKM/pages/459309751/Configurazione+API+Payment+Manager+su+APIM+pagoPA

File: src/apim_payment_manager.tf [hunk 4]
```diff
@@ -0,0 +1,541 @@
...
+}
+
+############################
+## API admin panel        ##
+############################
+locals {
+  apim_pm_adminpanel_api = {
+    display_name          = "Payment Manager admin panel frontend"
+    description           = "Frontend to support PM operations"
+    path                  = "payment-manager/admin-panel"
```
Comment (pasqualespica) [inline src/apim_payment_manager.tf:440]: update https://pagopa.atlassian.net/wiki/spaces/TKM/pages/459309751/Configurazione+API+Payment+Manager+su+APIM+pagoPA

File: src/apim_payment_manager.tf [hunk 5]
```diff
@@ -0,0 +1,541 @@
...
+}
+
+#####################
+## API wisp        ##
+#####################
+locals {
+  apim_pm_wisp_api = {
+    display_name          = "Payment Manager Wisp"
+    description           = "Frontend to support payments"
+    path                  = "payment-manager/wisp"
```
Comment (pasqualespica) [inline src/apim_payment_manager.tf:495]: update https://pagopa.atlassian.net/wiki/spaces/TKM/pages/459309751/Configurazione+API+Payment+Manager+su+APIM+pagoPA

**Review by pasqualespica — DISMISSED** [review]
great 👍

**Review by uolter — APPROVED** [review]

File: src/apim_payment_manager.tf (hunk 1 above)
Comment (infantesimone) [reply src/apim_payment_manager.tf:109]: in order to descrive the API version currently active for `restapi`, so only v4

**Pull request merged**